sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -60,13 +60,13 @@ export async function executeAtomXmlOperation(
     }
   } catch (err) {
     const error = new RestError(
-      `ResponseNotInAtomXMLFormat - ${err.message}`,
+      `Response not in Atom XML format`,
```

Comment:
Can we have this error message and the others that are thrown when parsing service output use similar verbiage?
The first sentence can be common and the second can provide more details.

Using the text from the serviceBusAtomManagementClient.ts error message, this can be

> Response from service is in bad format. Expected response to be in Atom XML format.

?

cc @bterlson

Reply:
Updated